lib/fsm/utils.go

```diff
@@ -41,9 +41,42 @@ func CanRollback(plan *storage.OperationPlan, phaseID string) error {
 		return trace.BadParameter(
 			"phase %q has already been rolled back", phase.ID)
 	}
+	if !dryRun && !latestRollback(plan.Phases, phaseID) {
```

Comment:
Rolling back an operation as a dry run does not change the state of operation phases. So dry run rollbacks would always fail this check. For now, simple workaround was to just ignore this check during dry runs.